spring-cloud-stream-binder-rabbit-core/pom.xml

```diff
@@ -35,6 +35,12 @@
 			<groupId>com.rabbitmq</groupId>
 			<artifactId>http-client</artifactId>
 			<version>2.0.1.RELEASE</version>
+			<exclusions>
+				<exclusion>
+          			<groupId>io.projectreactor.ipc</groupId>
+          			<artifactId>reactor-netty</artifactId>
```

Comment:
That's wrong direction.
You definitely need to upgrade to the `http-client.2.1.0.RELEASE` where all those guys are `optional` already: https://github.com/rabbitmq/hop/releases

Reply:
Good point Artem. Will update shortly